### DISCUSSION TITLE
python interface to wasptool repl ?

### DISCUSSION THREAD

**Discussion by tigercoding56** [wasp-os/wasp-os#399] [Q&A] [answered]
is there a way to interface to the wasptool micropython repl using a python library (i am trying to use the Level app as game controller (i modified it to print out orientation of the watch and plan to send keystrokes to a game if orientation of watch exceeds 35 degrees for example )     )

**Comment by thiswillbeyourgithub** [comment, marked as answer]
Take a look at tools/wasptool --console argument. That should help you!

**Reply by tigercoding56** [reply]
ok   
ty
